net/discovery.go

```diff
@@ -196,3 +217,17 @@ func (d *discovery) discover(provides types.ProvidesCoin,
 		}
 	}
 }
+
+func (d *discovery) discover(
+	provides types.ProvidesCoin,
+	searchTime time.Duration,
+) ([]peer.AddrInfo, error) {
+	log.Debugf("attempting to find DHT peers that provide [%s] for %vs...",
```

Comment:
Normally, an ellipsis at the end of a log message message is used to indicate that there will be a matching log message after the long operation completes.

Reply:
updated